net-fs/samba/samba-4.13.7.ebuild

```diff
@@ -86,7 +86,14 @@ COMMON_DEPEND="
 			net-dns/bind-tools[gssapi]
 		)
 	")
-	!alpha? ( !sparc? ( sys-libs/libunwind:= ) )
+	!alpha? (
+		!sparc? (
+			|| (
+				sys-libs/libunwind:=
+				sys-libs/llvm-libunwind:=
```

Comment:
These dependencies should be:
```
	!alpha? (
		llvm-libunwind? ( sys-libs/llvm-libunwind:0= )
		!llvm-libunwind? ( sys-libs/libunwind:0= )
	)
```
https://bugs.gentoo.org/779742 exists for keywording of sys-libs/llvm-libunwind on MIPS/PPC32/SPARC. (So you may need to wait a while...)